keps/sig-api-machinery/2896-openapi-v3/README.md

```diff
@@ -547,28 +585,40 @@ It's impossible to provide comprehensive guidance, but at the very high level
 These goals will help you determine what you need to measure (SLIs) in the next
 question. -->
 
+This feature should not affect the SLO of any components.
+
+The size of the OpenAPI scales as a cluster has more object types. In OpenAPI V2, aggregating the OpenAPI for all types was very resource intensive. In OpenAPI V3, aggregation is split and the aggregator will act as a proxy instead. Additional computation will be taken to build the OpenAPI V3 specs, and CRD specs will be aggregated after being built. We will monitor the current metrics to ensure that no regression is introduced with OpenAPI V3.
```

Comment:
this would be more precise: 

OpenAPI v3 aggregation comes on-top of v2 aggregation => there is additional load. But v3 aggregation is cheap:
1. OpenAPI v3 aggregation is much cheaper a every group-version is aggregated independently.
2. APIServices (aggregated apiservers) coming and going (due to availability changes) do not lead to aggregation because kube-apiserver only acts as a proxy
3. CRD aggregation is structurally trivial (no unification of definition names, which is quadratic in schema sizes) and hence linear in the number of CRDs per group-version.